### PR TITLE
Introduce sub-head for easier linking + cross-linking + moving around text

### PR DESCRIPTION
**Pull request by udidahan**

### CODE REVIEW

**Comment by Court72** [comment]
@msimecek 

Can you review the proposed changes? *IMPORTANT*: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Review by msimecek — APPROVED** [review]

**Comment by msimecek** [comment]
#sign-off

File: docs/reference-architectures/containers/aks-mission-critical/mission-critical-data-platform.md [hunk 1]
```diff
@@ -155,18 +155,23 @@ Azure Service Bus premium tier is the recommended solution for high-value messag
...
   - If the consumer successfully processes the message, it sends an acknowledgment back to the broker, and the message is removed from the queue.
   - If an acknowledgment isn't received by the broker in the allotted time period, or the handler explicitly abandons the message, the exclusive lock is released. The message is then available for other consumers to process the message.
-  - If a message is not successfully processed a configurable number of times, or the handler forwards the message to the dead-letter queue.
+  - If a message is not successfully processed a configurable number of times, or the handler forwards the message to the [dead-letter queue](/azure/service-bus-messaging/service-bus-dead-letter-queues).
+    - To ensure that messages sent to the dead-letter queue are acted upon, the dead-letter queue should be monitored, and alerts should be set.
+    - The system should have tooling for operators to be able to [inspect, correct, and resubmit messages](/azure/service-bus-messaging/service-bus-dead-letter-queues?source=recommendations#sending-dead-lettered-messages-to-be-reprocessed).
```
Comment (ckittel) [inline docs/reference-architectures/containers/aks-mission-critical/mission-critical-data-platform.md:160]:
```suggestion
    - The system should have tooling for operators to be able to [inspect, correct, and resubmit messages](/azure/service-bus-messaging/service-bus-dead-letter-queues#sending-dead-lettered-messages-to-be-reprocessed).
```

**Review by ckittel — APPROVED** [review]

**Comment by ckittel** [comment]
#sign-off

**Review by ckittel — APPROVED** [review]

**Review by Court72 — APPROVED** [review]

**Comment by udidahan** [comment]
Wonderful. Thank you.

**Pull request merged**